### DISCUSSION TITLE
How can I add URL based versioning?

### DISCUSSION THREAD

**Discussion by rbambrough-intel** [jasontaylordev/CleanArchitecture#1106] [Ideas]
I want to be able to have different versions of the endpoints, however I am having difficulty modifying the template to support it.

Is there a recommended way of doing it?

**Comment by jasontaylordev** [comment]
You need to dig deeper to determine the best approach for your goal. If you're new to Minimal API, start with a [review of the documentation](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-8.0) and ensure you have a good understanding of how it works outside of the template. Then, within the template, review the sample endpoints and Minimal API extensions within [src/Web/Infrastructure](https://github.com/jasontaylordev/CleanArchitecture/tree/main/src/Web/Infrastructure).
